Libraries/Text/TextInput/RCTBaseTextInputViewManager.m

```diff
@@ -52,7 +52,6 @@ @implementation RCTBaseTextInputViewManager
 RCT_REMAP_OSX_VIEW_PROPERTY(spellCheck, backedTextInputView.automaticSpellingCorrectionEnabled, BOOL) // TODO(macOS GH#774)
 RCT_REMAP_NOT_OSX_VIEW_PROPERTY(caretHidden, backedTextInputView.caretHidden, BOOL) // TODO(macOS GH#774)
 RCT_REMAP_NOT_OSX_VIEW_PROPERTY(clearButtonMode, backedTextInputView.clearButtonMode, UITextFieldViewMode) // TODO(macOS GH#774)
-RCT_REMAP_VIEW_PROPERTY(scrollEnabled, backedTextInputView.scrollEnabled, BOOL)
```

Comment:
`scrollEnabled` is only relevant for multiline input. Moved the remap to `RCTMultilineTextInputViewManager`

Reply:
Is this a change to the iOS implementation as well? Does that need to be made upstream first or afterwards?